include/swift/AST/Types.h

```diff
@@ -3750,6 +3774,16 @@ class SILFunctionType final : public TypeBase, public llvm::FoldingSetNode,
     // Is this function guaranteed to be no-escape by the type system?
     bool isNoEscape() const { return Bits & NoEscapeMask; }
 
+    bool isDifferentiable() const {
+      return getDifferentiabilityKind() >
```

Comment:
nit, but it seems like != would be more natural here than >

Reply:
Fixed